### ISSUE TITLE
`TabList` does not show focus indicator in high contrast mode

### ISSUE THREAD

**Issue by ling1726** [microsoft/fluentui#27122]
Repro: https://codesandbox.io/s/zm20so?file=/example.tsx

In windows high contrast mode there is no focus indicator on tabs

![image](https://user-images.githubusercontent.com/20744592/223671943-0f510b96-bb52-4aa8-8fa5-52fa071ef684.png)
